packages/owl-bot/test/scan-googleapis-gen-and-create-pull-requests.ts

```diff
@@ -494,12 +500,16 @@ Copy-Tag: ${copyTagFrom('.github/.OwlBot.yaml', abcCommits[2])}`;
     assert.strictEqual(gitLog, `${commitMessage2}\n\n${commitMessage1}\n\n`);
 
     // Confirm the pull request body was updated.
+    // The first line of commitMessage2 becomes PR title.
+    const newline = commitMessage2.indexOf('\n');
+    const body2 = commitMessage2.slice(newline).trim();
     assert.deepStrictEqual(pulls.updates, [
       {
-        body: `${EMPTY_REGENERATE_CHECKBOX_TEXT}\n\n${commitMessage2}\n\n${commitMessage1}\n\n`,
+        body: `${EMPTY_REGENERATE_CHECKBOX_TEXT}\n\n${body2}\n\n${commitMessage1}\n\n`,
```

Comment:
nit: it might be nice to use something like `snap-shot-it` to record the full message. We're building the expected output so reading the test is hard to see what the final output looks like.

Reply:
I tried, but the problem is the copy-tag varies with the source repo commit hash, and the source repo commit hash varies every time the test runs.